docs/platforms/react-native/integrations/default.mdx

```diff
@@ -3,4 +3,214 @@ title: Default Integrations
 description: "Learn more about system integrations: Dedupe, FunctionToString, Breadcrumbs, LinkedErrors, and HttpContext (UserAgent), which are integrated into the standard library or the interpreter itself by default."
 ---
 
-<Include name="platforms/configuration/integrations/default.mdx" />
+The below system integrations are part of the standard library or the interpreter itself and are enabled by default. To understand what they do and how to disable them if they cause issues, read on.
```

Comment:
I copied this here as it was only used here, as far as I can tell @krystofwoldrich . Eventually we'll need to update the integrations for react-native as well, I left them as-is for now!